test/integration/test_ssh_access.py

```diff
@@ -1,12 +1,29 @@
+import contextlib
 import fabric
 import io
 import os
 import pytest
 
-from .conftest import container_named
+from exasol_integration_test_docker_environment.lib.docker import ContextDockerClient
 from exasol_integration_test_docker_environment.lib.base.ssh_access import SshKey, SshKeyCache
 
 
+def exact_matcher(names):
```

Comment:
I moved that to a helper module, in my previous PR

Reply:
https://github.com/exasol/integration-test-docker-environment/blob/main/test/integration/helpers.py